application/utils/noise_filter/schemas.py

```
@@ -0,0 +1,155 @@
+
+
+__all__ = [
+    "ChangeRecord",
+    "Source",
+    "GithubSource",
+    "RssSource",
+    "Span",
+    "Locator",
+    "ClassifyResult",
+    "QueuePayload",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)